frontend/src/clients/ReportClient.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpClient } from '@src/clients/Httpclient'
+import { AxiosError, HttpStatusCode } from 'axios'
+import { InternalServerException } from '@src/exceptions/InternalServerException'
+import { reportResponseProps } from '@src/types/reportResponse'
+
+export class ReportClient extends HttpClient {
+  reportResponse: reportResponseProps = {
+    velocity: {
+      velocityForSP: '',
+      velocityForCards: '',
+    },
+  }
+
+  generateReporter = async () => {
+    try {
+      await this.axiosInstance.post(`/report`).then((res) => {
```

Comment:
Is there any query for `/report` request?

Reply:
The current request does not have params

Reply:
will it have query in the future? Otherwise, should we change to get?